### PR TITLE
Add log collection instruction for docker

### PR DESCRIPTION
**Pull request by NilsBunge**
### What does this PR do?
Improve the read.me to include instruction for log collection

### Motivation
Log collection is available in Docker since agent 6, therefore it should be documented.

### CODE REVIEW

File: Dockerfiles/agent/README.md [hunk 1]
```diff
@@ -30,3 +30,59 @@ You'll need to download one of the `datadog-agent*_amd64.deb` package in this di
...
+
+To  run a Docker container which embeds the Datadog Agent to monitor your host use the following command:
+
+```
+docker run -d --name dd-agent -h `hostname` -e DD_API_KEY=<YOUR_API_KEY>  -e DD_LOG_ENABLED=true -e SD_BACKEND=docker -v /var/run/docker.sock:/var/run/docker.sock:ro -v /proc/:/host/proc/:ro -v /opt/datadog-agent/run:/opt/datadog-agent/run:rw -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro -v /opt/datadog-agent/conf.d:/conf.d:ro datadog/agent:6.0.0-beta.6
+```
+
+*Important notes*: 
+
+- The Docker integration is enabled by default, as well as [autodiscovery](https://docs.datadoghq.com/guides/servicediscovery/) in auto config mode (remove the `SD_BACKEND` variable to disable it).
```
Comment (xvello) [inline Dockerfiles/agent/README.md]: the `SD_BACKEND` variable is not supported by agent6, to disable docker autodiscovery, one must change `datadog.yaml` and remove the `listeners: -docker` section. For now we don't provide any envvar binding as that'll probably be overhauled before RC

File: Dockerfiles/agent/README.md [hunk 2]
```diff
@@ -30,3 +30,59 @@ You'll need to download one of the `datadog-agent*_amd64.deb` package in this di
...
+
+- For this example we used the image 6.0.0-beta.6 of the agent. You can find [here](https://hub.docker.com/r/datadog/agent/tags/) the latest available images for agent 6 and we encourage you to always pick the latest version
```
Comment (xvello) [inline Dockerfiles/agent/README.md]: Starting with `beta7`, to be out in a few hour, we'll be providing the `latest` tag, could you update accordingly?

File: Dockerfiles/agent/README.md [hunk 3]
```diff
@@ -30,3 +30,59 @@ You'll need to download one of the `datadog-agent*_amd64.deb` package in this di
...
+The Datadog Agent can collect logs from containers starting at the version 6. Two installations are possible:
+ 
+- on the host: where the agent is external to the Docker environment 
+- or by deploying its containerized version in the Docker environment
+
+### Setup
+
+First let’s create two directories on the host that we will later mount on the containerized agent:
+
+- `/opt/datadog-agent/run`: to make sure we do not lose any logs from containers during restarts or network issues we store on the host the last line that was collected for each container in this directory
```
Comment (xvello) [inline Dockerfiles/agent/README.md]: Note that docker does not need the folders to be created beforehand (it'll create them automatically), but let's keep it step-by-step for now

File: Dockerfiles/agent/README.md (hunk 2 above)
Comment (NBParis) [reply Dockerfiles/agent/README.md]: @xvello Thanks for the update, that's an awesome news. i'll make the update

File: Dockerfiles/agent/README.md (hunk 1 above)
Comment (NBParis) [reply Dockerfiles/agent/README.md]: 👍  updated

File: Dockerfiles/agent/README.md (hunk 2 above)
Comment (NBParis) [reply Dockerfiles/agent/README.md]: 👍  updated

**Review by xvello — APPROVED** [review]
🚢

**Pull request merged**